app/assets/javascripts/discourse/app/instance-initializers/mobile-keyboard.js

```diff
@@ -66,7 +66,8 @@ export default {
     } else {
       let viewportWindowDiff =
         this.windowInnerHeight - window.visualViewport.height;
-      if (viewportWindowDiff > 0) {
+      const FLOATING_IPAD_KEYBOARD_HEIGHT = 71.5;
```

Comment:
this is a bug of our current implementation to consider a small toolbar as a full blown keyboard. I don't think we need to handle this small keyboard, if we ever had to, we should create a dedicated class to add to the body.